publish/dir.targets

```diff
@@ -62,8 +62,10 @@
     <ItemGroup>
       <_FoundBlobNames Include="%(_BlobNames.FileName)%(_BlobNames.Extension)" Condition="'%(_BlobNames.Extension)' == '.svg'" />
       <_MissingBlobNames Include="@(PublishRid->'sharedfx_%(Identity)_$(ConfigurationGroup)_version_badge.svg')" 
-                          Exclude="@(_FoundBlobNames)" />
+                         Exclude="@(_FoundBlobNames)" />
     </ItemGroup>
+    <Error Text="Missing publish output from all legs of the build. Missing blobs are '@(_MissingBlobNames)'"
```

Comment:
This is a duplicate of the message in publish.proj.  Having it here is fine, but make it `Importance=High`, and remove it from publish.proj